ingest/core/ingest.py

```
@@ -0,0 +1,210 @@
+        """
+        supported_extensions = {".pdf", ".md", ".txt"}
+        documents = []
+
+        for root, dirs, files in os.walk(corpus_path):
+            for file in files:
+                file_path = Path(root) / file
+                if file_path.suffix.lower() in supported_extensions:
+                    documents.append(file_path)
+
+        print("Discovered {len(documents)} documents:")
+        for doc in sorted(documents):
```

Ruff (0.13.1)

Loop control variable `doc` not used within loop body

Rename unused `doc` to `_doc`

(B007)